pyspecProcScripts/integrate_limits.py

```diff
@@ -37,9 +37,7 @@ def integrate_limits(s, axis="t2", fl=None):
         fl.plot(signal_E['sigma':(filter_width,filter_width+1e-6)],'o', human_units=False)
         fl.pop_marker()
     fwhm = filter_width
-    fl.push_marker()
     temp.mean_all_but(axis)
     # https://en.wikipedia.org/wiki/Full_width_at_half_maximum
     temp.convolve(axis, fwhm/(2*np.sqrt(np.log(4))))
-    fl.pop_marker()
```

Comment:
these were just extra + actually caused a bug.

Reply:
(b/c they weren't inside an `if fl is None` conditional)

Reply:
okay so when using push_marker/pop_marker they need to be inside an if 'fl is not None:' instance?

Reply:
yes